app/assets/javascripts/line_items/update_quantity.js

```diff
@@ -0,0 +1,3 @@
+updateQuantity = function(lineItemId) {
```

Comment:
@b-artem This function is defined in global scope. Let's isolate it like you did for other pages

Reply:
Done